assets/js/base/components/cart-checkout/shipping-rates-control/style.scss

```diff
@@ -34,31 +34,8 @@
 	margin-bottom: 0;
 }
 
-// Resets when it's inside a panel.
 .wc-block-shipping-rates-control {
-	.wc-block-shipping-rates-control__package.components-panel__body {
```

Comment:
The class `.components-panel__body` was used by the `Panel` component from `@wordpress/components`. Since we migrated to our own `Panel` component which has different classes, all the code below was dead code and can be safely removed.